src/utils/crypto.js

```diff
@@ -13,8 +13,8 @@ export const outputValue = (currentUtxo, sendFeeRate, price) =>
 
 export function calculateFee({ vins, vouts, recommendedFeeRate, includeChangeOutput = true }) {
     const baseTxSize = 10;
-    const inSize = 180;
-    const outSize = 34;
+    const inSize = 57.5;
+    const outSize = 43;
```

Comment:
nice, though we should do `Math.round(fee)` on line 22 so we ensure we return an int

Reply:
and lets make a note in the comments to follow up with exact calculations (not super important currently)

Reply:
done

Reply:
line 26 i mean

Reply:
yep yep